src/segments/zig_test.go

```diff
@@ -0,0 +1,100 @@
+package segments
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/jandedobbeleer/oh-my-posh/src/runtime"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestZig(t *testing.T) {
+	cases := []struct {
+		Case                  string
+		Version               string
+		InProjectDir          bool
+		ExpectedString        string
+		ExpectedMajor         string
```

Comment:
I don't think these are used below (any of the individual version strings), so they can be removed.